libs/pages/clusters/src/lib/ui/page-clusters-create/step-features/step-features.tsx

```diff
@@ -25,12 +28,86 @@ export interface StepFeaturesProps {
   goToBack?: () => void
 }
 
+export function checkSubnetsNotEmpty(subnets?: Subnets[]): boolean {
```

Comment:
This think it should be easier turn that into `areSubnetsEmpty`

Reply:
https://github.com/Qovery/console/pull/1211/commits/3d7d75ea31a22be0f14ca63d48d2330af4f45100